modules/synthetic-tokens/src/lib.rs

```diff
@@ -214,6 +214,6 @@ impl<T: Trait> LiquidityPoolManager<T::LiquidityPoolId, T::Balance> for Module<T
 	}
 
 	fn ensure_can_withdraw(_pool: T::LiquidityPoolId, _amount: T::Balance) -> DispatchResult {
-		unimplemented!()
```

Comment:
`ensure_can_withdraw` of synthetic tokens is not used no?

Reply:
Just because integration failed.
```
thread 'tests::test_margin_liquidity_pools_works' panicked at 'not yet implemented', modules/synthetic-tokens/src/lib.rs:217:3
```

Reply:
It's more reasonable to find out if `ensure_can_withdraw` is called accidentally.

Reply:
Additionally, please includes these fixes in your integration tests PR instead, to provide context of the problem.